tests/cases/docker/pyright/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:current
+RUN git clone https://github.com/microsoft/pyright.git /pyright
+WORKDIR /pyright
+RUN git pull
+RUN npm i
+COPY --from=typescript/typescript /typescript/typescript-*.tgz /typescript.tgz
+RUN npm install /typescript.tgz --exact --ignore-scripts --save-dev
+RUN npx lerna exec --stream --concurrency 1 -- npm install /typescript.tgz --exact --ignore-scripts --save-dev
+WORKDIR /pyright/packages/pyright
+ENTRYPOINT [ "npx" ]
+CMD [ "webpack", "--mode", "production" ]
```

Comment:
Their normal build command is `npm run build`, which desugars to `webpack --mode production --progress` - but the `--progress` flag adds a bunch of log output we have zero interest in; so we invoke `webpack` directly without it. This is more fragile than using their build task, though.

Reply:
Hm, if only there was a way to disable explicit flags after the fact. This is probably okay though, right @jakebailey?

Reply:
Would it be better for you to just run `npx lerna exec --stream --no-bail -- tsc --noEmit`? This is what our PR/validation pipeline does, just to make sure things typecheck (before moving on to doing a full build).

I think you can double-declare the flag by running `npm run build --progress=false`, but honestly anything you put here is going to end up being fragile as it depends on our build process.

Reply:
Note that we have more than just this entrypoint; `vscode-pyright` is its own binary too, though I suppose it is much smaller.